components/match2/wikis/trackmania/match_group_input_custom.lua

```diff
@@ -0,0 +1,538 @@
+---
+-- @Liquipedia
+-- wiki=trackmania
+-- page=Module:MatchGroup/Input/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local CustomMatchGroupInput = {}
+
+local Json = require('Module:Json')
+local Logic = require('Module:Logic')
+local Array = require('Module:Array')
+local Lua = require('Module:Lua')
+local PageVariableNamespace = require('Module:PageVariableNamespace')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+local Template = require('Module:Template')
+local TypeUtil = require('Module:TypeUtil')
+local Variables = require('Module:Variables')
+local getIconName = require('Module:IconName').luaGet
+local Streams = require('Module:Links/Stream')
+
+local MatchGroupInput = Lua.import('Module:MatchGroup/Input', {requireDevIfEnabled = true})
+local Opponent = Lua.import('Module:Opponent', {requireDevIfEnabled = true})
+
+local _STATUS_HAS_SCORE = 'S'
```

Comment:
the naming of the "global" locals is not consistent 
either prefix everything with underscore or nothing (i prefer the later)

Reply:
https://github.com/Liquipedia/Lua-Modules/pull/2259/commits/ba3655ba45f813db59a0f6545fc944d76ff6d285